libs/auth-api-lib/src/lib/delegations/models/delegation-delegation-type.model.ts

```diff
@@ -0,0 +1,61 @@
+import type {
+  CreationOptional,
+  InferAttributes,
+  InferCreationAttributes,
+} from 'sequelize'
+import {
+  Table,
+  Column,
+  Model,
+  ForeignKey,
+  DataType,
+  CreatedAt,
+  UpdatedAt,
+  BelongsTo,
+} from 'sequelize-typescript'
+
+import { Delegation } from './delegation.model'
+import { DelegationTypeModel } from './delegation-type.model'
+
+@Table({
```

Comment:
You can and should add the delegation delegation_type unique constraint in the `@Table` decorator, so tests run with the same constraints as prod.